app/views/eligibility_interface/qualified_for_subject/new.html.erb

```diff
@@ -0,0 +1,56 @@
+<% content_for :page_title, "#{'Error: ' if @qualified_for_subject_form.errors.any?}Does your teaching qualification or university degree qualify you to teach one of the following subjects?" %>
+<% content_for :back_link_url, back_link_url(eligibility_interface_teach_children_path) %>
+
+<h1 class="govuk-heading-l">Does your teaching qualification or university degree qualify you to teach one of the following subjects?</h1>
+<p class="govuk-hint">
+  You'll need to upload your certificate, along with a transcript that shows all the modules you studied and the scores you achieved.
+</p>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Maths</li>
+      <li>Science</li>
+      <li>Biology</li>
+      <li>Chemistry</li>
+      <li>Physics</li>
+      <li>French</li>
+      <li>German</li>
+    </ul>
+  </div>
+
+  <div class="govuk-grid-column-one-half">
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Italian</li>
+      <li>Japanese</li>
+      <li>Latin</li>
+      <li>Mandarin</li>
+      <li>Russian</li>
+      <li>Spanish</li>
+    </ul>
+  </div>
+</div>
+
+<%= form_with model: @qualified_for_subject_form, url: eligibility_interface_qualified_for_subject_url, method: :post do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <%= f.govuk_collection_radio_buttons(
+    :qualified_for_subject,
+    [OpenStruct.new(label: 'Yes', value: true), OpenStruct.new(label: 'No', value: false)],
+    :value,
+    :label,
+    legend: nil,
+  ) %>
+  <%= f.govuk_submit prevent_double_click: false %>
+<% end %>
+
+<details class="govuk-details" data-module="govuk-details">
```

Comment:
I'd suggest using the [`govuk_details`](https://govuk-components.netlify.app/components/details/) component here.